frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -50,7 +50,7 @@ export function SurveyForm({ id }: { id: string }): JSX.Element {
     const { loadSurvey, editingSurvey } = useActions(surveyLogic)
 
     return (
-        <Form formKey="survey" logic={surveyLogic} className="space-y-4" enableFormOnSubmit>
+        <Form id="survey" formKey="survey" logic={surveyLogic} className="space-y-4" enableFormOnSubmit>
```

Comment:
@mariusandra Do you think it'd make sense for `kea-forms` to add an ID automatically based on logic key + `formKey`? Or would that be annoying?